frontend/src/store/screenSizeState.ts

```diff
@@ -0,0 +1,20 @@
+import { sideNavState } from "@/store/SideNavState";
+import { setNotificationMsg, newMsgList, newFormData, newSeverity } from "./NotificationState";
+
+export let screenSize = window.innerWidth;
```

Comment:
Hi Jota, if you export this variable and share it with other components, it has to be a reactive variable, not a local changeable variable.  Cause whenever you change its value you want all other components receive the new value and re-render the page.

Reply:
Sure!

Reply:
Use getter if you need screenSize value.